docs/guides/protect-1-service-network-policies.mdx

```diff
@@ -277,7 +277,7 @@ otterize network-mapper export --server productcatalogservice.otterize-ecom-demo
 | kubectl apply -f -
 ```
 
-If we now look at the access graph, lo and behold: the lines and arrows from all 3 clients to `productcatalogservice` are now **solid** green: all the necessary intents have been declared. (And a lot of other lines are now green too, since these clients also call many other servers; these other servers may soon be ready to protect too.)
+If we now look at the access graph, lo and behold: the lines and arrows from all 3 clients to `productcatalogservice` are now  green: all the necessary intents have been declared. (And a lot of other lines are now green too, since these clients also call many other servers; these other servers may soon be ready to protect too.)
```

Comment:
```suggestion
If we now look at the access graph, lo and behold: the lines and arrows from all 3 clients to `productcatalogservice` are now green: all the necessary intents have been declared. (And a lot of other lines are now green too, since these clients also call many other servers; these other servers may soon be ready to protect too.)
```

Reply:
👍